eks/outputs.tf

```diff
@@ -31,3 +31,13 @@ output "nomad_server_key" {
 output "nomad_tls_ca" {
   value = module.nomad.nomad_tls_ca
 }
+
+locals {
+  bastion_connect_command = var.bastion_key == "" ? "mssh ubuntu@${aws_instance.bastion[0.0].id}" : "ssh ubuntu@${aws_instance.bastion[0.0].public_ip}"
```

Comment:
Should both sides of this if statement use the same command?

Reply:
They don't: One side uses `mssh` with the `instance-id` and the other side uses `ssh` with the public IP